rust/src/transformations/polars/write_csv/mod.rs

```diff
@@ -0,0 +1,85 @@
+use std::path::PathBuf;
+
+use polars::prelude::*;
+
+use crate::{
+    core::{Function, Metric, MetricSpace, StabilityMap, Transformation},
+    domains::{CsvDomain, DataFrameDomain},
+    error::Fallible,
+    metrics::IntDistance,
+};
+
+/// Make a Transformation that writes a DataFrame into a CSV file.
+///
+/// # Arguments
+/// * `input_domain` - DataFrameDomain of the data to be written into a file.
+/// * `input_metric` - Metric of the data type to be written into a file.
+/// * `path` - Path to the output file.
+pub fn make_write_csv<M: Metric<Distance = IntDistance>>(
+    input_domain: DataFrameDomain,
+    input_metric: M,
+    path: PathBuf,
+) -> Fallible<Transformation<DataFrameDomain, CsvDomain<DataFrame>, M, M>>
+where
+    (CsvDomain<DataFrame>, M): MetricSpace,
+    (DataFrameDomain, M): MetricSpace,
+{
+    let output_domain = CsvDomain::new(input_domain.clone());
+    Transformation::new(
+        input_domain.clone(),
+        output_domain.clone(),
+        Function::new_fallible(move |data_frame: &DataFrame| {
+            output_domain
+                .new_writer(path.clone())?
+                .finish(&mut data_frame.clone())?;
+            Ok(path.clone())
+        }),
+        input_metric.clone(),
+        input_metric.clone(),
+        StabilityMap::new_from_constant(1),
+    )
+}
+
+#[cfg(test)]
+mod test_make_sink_csv {
+    use crate::domains::{AtomDomain, LazyFrameDomain, SeriesDomain};
+    use crate::error::ExplainUnwrap;
+    use crate::metrics::SymmetricDistance;
+    use crate::transformations::make_collect;
+
+    use super::*;
+
+    #[test]
+    fn test_make_sink_csv() -> Fallible<()> {
+        let lazyframe_domain = LazyFrameDomain::new(vec![
+            SeriesDomain::new("A", AtomDomain::<i32>::new_closed((1, 4))?),
+            SeriesDomain::new("B", AtomDomain::<f64>::new_closed((0.5, 5.5))?),
+        ])?
+        .with_counts(df!["A" => [1, 2], "count" => [1, 2]]?.lazy())?
+        .with_counts(df!["B" => [1.0, 2.0], "count" => [2, 1]]?.lazy())?;
+
+        let lazy_frame = df!(
+            "A" => &[1, 2, 2],
+            "B" => &[1.0, 1.0, 2.0],)?
+        .lazy();
+
+        let mut path = PathBuf::new();
+        path.push(
+            r"/Users/pediatria/Desktop/Harvard/openDP/opendp/rust/src/transformations/tmp.csv",
```

Comment:
This path doesn't exist on CI. How about using this crate as a dev-dependency:
https://docs.rs/tempdir/latest/tempdir/struct.TempDir.html

Reply:
sorry, the test was not supposed to be included in the commit. Do we want to keep it with the temp dir approach? I would add similar test in parquet PR too

Reply:
Yeah how about you remove it from the rust test, and just use the Python test, where the batteries are already loaded. Avoids adding another dependency.